src/modules/shared/components/ThemeToggle/ThemeToggle.tsx

```diff
@@ -0,0 +1,28 @@
+import { useContext } from "react";
+import { ThemeContext } from "../../../../context/ThemeContext";
+import styles from "./ThemeToggle.module.css";
+
+const ThemeToggle = () => {
+  const context = useContext(ThemeContext);
+
+  if (context === undefined) {
+    throw new Error("ThemeContext is undefined");
+  }
+
+  const { theme, toggleTheme } = context;
+  const handleClick = () => {
+    toggleTheme();
+  };
+
+  return (
+    <div onClick={handleClick} className="rounded ">
```

Comment:
This should be a button, we can also add a descriptive text with sr-only class for screen readers.